nubis/files/confd/templates/secrets_xmatters.tmpl

```diff
@@ -1,6 +1,8 @@
 config = {
-  'proxies'      : {"https" : "http://proxy.service.consul:3128"},
-  'xm_client_id' : '{{ getv "/config/XMatters/ClientID" "xmatters-clientid-not-set" }}',
-  'xm_username'  : '{{ getv "/config/XMatters/Username" "xmatters-username-not-set" }}',
-  'xm_password'  : '{{ getv "/config/XMatters/Password" "xmatters-password-not-set" }}',
+  'proxies'       : {"https" : "http://proxy.service.consul:3128"},
+  'xm_client_id'  : '{{ getv "/config/XMatters/ClientID" "xmatters-clientid-not-set" }}',
+  'xm_username'   : '{{ getv "/config/XMatters/Username" "xmatters-username-not-set" }}',
+  'xm_password'   : '{{ getv "/config/XMatters/Password" "xmatters-password-not-set" }}',
+  'url'           : '{{ getv "/config/XMatters/Url" "https://mozilla-np.xmatters.com" }}',
+  'supervisor_id' : '{{ getv "/config/XMatters/SupervisorID" "72a77545-4c4b-465d-b22a-41a14e0a1b78" }}',
```

Comment:
Is this a sensible default value ? looks like a UUID, might better belong just in configuration no ?

Reply:
shrug. it's the stage value. I figured that would work for a default.